spanner/setup.py

```diff
@@ -51,12 +51,14 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.27.0, < 0.28dev',
-    'google-auth >= 1.1.0',
-    'grpcio >= 1.2.0, < 2.0dev',
```

Comment:
Are we allowing our dependencies to pick the right `grpcio`?

Reply:
Yep, `google-gax` (eventually `google-api-core`) will be the point of truth on the correct gRPC. This is increasingly important in a world where things like grpc/grpc#12455 happen.